test/imprimatur.py

```diff
@@ -6517,12 +6517,11 @@ def virtual_bill(supply_source):
         'path': '/hhdc_contracts/34/auto_importer',
         'tries': {},
         'regexes': [
-            r"Finished loading",
```

Comment:
the test scripts failed. I compared it with master and there's no finished loading in the auto importer.